app/models/tag_adjustment.rb

```diff
@@ -1,6 +1,5 @@
 class TagAdjustment < ApplicationRecord
-  validates :tag_name, presence: true,
-                       uniqueness: { scope: :article_id, message: I18n.t("models.tag_adjustment.unique") }
```

Comment:
Remove the scope that created a uniqueness on the article_id and tag_name for a Tag Adjustment, thus allowing tag_moderators to override each other actions. Same applied for admins.